server/models/group.js

```diff
@@ -133,6 +133,17 @@ GroupSchema.statics = {
     });
   },
 
+  getGroups(query) {
+    return new Promise((resolve, reject) => {
```

Comment:
This looks very similar to the searchByName function above, is it only the regex you don't want here?